scm-ui/ui-components/src/repos/annotate/AnnotatePopover.tsx

```diff
@@ -50,8 +50,8 @@ const PopoverContainer = styled.div`
     top: 100%;
     left: 5.5em;
     border-color: transparent;
-    border-bottom-color: white;
-    border-left-color: white;
+    border-bottom-color: var(--scm-white-color);
+    border-left-color: var(--scm-white-color);
```

Comment:
> I think white isn't a very good name for a variable in a multi-theme environment. And the way it is used seems to prevent the styling of the border color of components like `Popover` or `NavigationBar` by theme.

I do not understand the issue mentioned. The definition specified that the element is white and so after the change. These components should never be stylable as far as I know.

> Changed to new css-variable var(--scm-popover-border-color).

Then it would probably be better to work globally with the SCSS variable `popover-border-color` you introduced and overwrite it accordingly, instead of defining a CSS variable twice. Currently the darkmode would contain both `:root`-definitions.